classes/ObjectModel.php

```diff
@@ -1197,6 +1207,10 @@ public function validateField($field, $value, $id_lang = null, $skip = [], $huma
             }
 
             $length = Tools::strlen($value);
+            if (isset($data['validate']) && Tools::strtolower($data['validate']) === 'isint') {
```

Comment:
Sorry I think I wasn't clear enough when I asked you about this modification but I think it may be risky to change the code responsible for `size` validation because it can have many unexpected side effects, even if the logic seems better for us this way we can't really anticipate the impact everywhere

So to be safer I was thinking of adding another option like `'range' => ['min' => 0, 'max' => 10]` This range option should be handled in ObjectModel in this same method so that any ObjectModel can use it And then we can update the `StockAvailable` definition and replace its `size` option with the new `range` option

The size validation is really thought for strings that's why I think having a dedicated validation option for numbers makes more sense Do you see the idea?